### PR TITLE
feat(INT-5829): create Google Container Analysis data model

### PR DESCRIPTION
**Pull request by gastonyelmini**
## Creates Google Container Analysis data model

For further understanding, please visit this link: https://whimsical.com/container-analysis-api-ingestion-LyWp3PCJrbYx7tHu4iqxox

Container Analysis API Ingestion

### CODE REVIEW

File: docs/spec/src/steps/services/containeranalysis/index.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    implemented: false,
+  },
+  {
+    /**
+     * PROPERTY: projects.noteOccurrences
+     * PATTERN: Build Child Relationships
+     * REQUIRED PERMISSIONS: n/a
+     */
+    id: 'build-container-analysis-note-occurrences-relationships',
+    name: 'Build Container Analysis -> Note Occurrence',
```
Comment (ndowmon) [inline docs/spec/src/steps/services/containeranalysis/index.ts:53]: nit: I think this is a slight typo?

```suggestion
    name: 'Build Container Analysis Note -> Occurrence',
```

**Review by ndowmon — APPROVED** [review]
Looks good! Thanks.

File: docs/spec/src/steps/services/containeranalysis/index.ts (hunk 1 above)
Comment (gastonyelmini) [reply docs/spec/src/steps/services/containeranalysis/index.ts]: Updated! Thank you

**Review by ndowmon — APPROVED** [review]

**Comment by gastonyelmini** [comment]
Closing because is not going to be implemented

**Pull request closed without merging**